build-edt.sh

```
@@ -15,12 +15,14 @@ if [ "${DOCKER_SYSTEM_PRUNE}" = 'true' ] ; then
 fi
 
 #Если версия EDT >= 2024.1.0, использовать JDK 17
-if [[ "$(printf "%s\n" "$EDT_VERSION" "2024" | sort -V | head -n 1)" == "2024" ]]; then
-  BASE_IMAGE="azul/zulu-openjdk"
-  BASE_TAG="17"
+if [[ "$EDT_VERSION" == 2024* ]] || [[ "$EDT_VERSION" == 2025* ]]; then
```

Shellcheck (0.11.0)

[info] Possible misspelling: EDT_VERSION may not be assigned. Did you mean edt_version?

(SC2153)